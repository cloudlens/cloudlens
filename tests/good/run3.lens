lens f(x, log){
    stream {
    	print(x + e.message)
    }
    {print("middle")}
    stream {
     	print(x + e.message)
    }
}

stream {
    print("0 " + e.message)
}

run f ("1 ")

stream {
    print("2 " + e.message)
}

run f ("3 ")

stream {
    print("4 " + e.message)
}
