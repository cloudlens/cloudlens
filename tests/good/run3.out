0 foo 1
1 foo 1
0 foo 2
1 foo 2
0 foo 3
1 foo 3
0 foo 4
1 foo 4
middle
1 foo 1
2 foo 1
3 foo 1
1 foo 2
2 foo 2
3 foo 2
1 foo 3
2 foo 3
3 foo 3
1 foo 4
2 foo 4
3 foo 4
middle
3 foo 1
4 foo 1
3 foo 2
4 foo 2
3 foo 3
4 foo 3
3 foo 4
4 foo 4
